notebooks/postgresql2bq/postgresql-to-bigquery-notebook.ipynb

```diff
@@ -0,0 +1,2413 @@
+    "import time\n",
+    "import copy\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "from google_cloud_pipeline_components.experimental.dataproc import DataprocPySparkBatchOp\n",
+    "import sqlalchemy\n",
+    "from sqlalchemy import text\n",
+    "import psycopg2\n",
+    "import math\n",
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "af0cf264-6eef-441d-a0a4-0c7bdc3d5b96",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Step 3: Assign Parameters\n",
+    "\n",
+    "## Step 3.1 Common Parameters\n",
+    "\n",
+    "PROJECT : GCP project-id\n",
+    "\n",
+    "REGION : GCP region\n",
+    "\n",
+    "GCS_STAGING_LOCATION : GCS staging location to be used for this notebook to store artifacts\n",
+    "\n",
+    "SUBNET : VPC subnet\n",
+    "\n",
+    "JARS : list of jars. For this notebook postgre connectora and postgres connectorjar is required in addition with the dataproc template\n",
+    "\n",
+    "MAX_PARALLELISM : Parameter for number of jobs to run in parallel default value is 5\n",
+    "\n",
+    "SERVICE_ACCOUNT: Custom service account email to use for vertex ai pipeline and dataproc job with above mentioned permissions\n",
+    "\n",
+    "POSTGRESQL_TO_BIGQUERY_JOBS : List of bigquery job IDs that will be created by Vertex AI pipelines to migrate data from source to BQ."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "fa46af62-62b2-4ad3-87e1-ff232f524e64",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "IS_PARAMETERIZED = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "c1ad4f98-8924-4ab9-8a7a-948cd82d1d0d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if not IS_PARAMETERIZED:\n",
+    "    GCP_PROJECT = \"\"\n",
+    "    REGION = \"\" # eg: us-central1 (any valid GCP region)\n",
+    "    GCS_STAGING_LOCATION = \"\" # eg: gs://my-staging-bucket/sub-folder\n",
+    "    SUBNET = \"projects/{project}/regions/{region}/subnetworks/{subnet}\"\n",
+    "    MAX_PARALLELISM = 5 # max number of tables which will migrated parallelly \n",
+    "    SERVICE_ACCOUNT = \"\" \n",
+    "    \n",
+    "# Do not change this parameter unless you want to refer below JARS from new location\n",
+    "JARS = [GCS_STAGING_LOCATION + \"/jars/postgresql-42.2.6.jar\", GCS_STAGING_LOCATION + \"/jars/spark-bigquery-with-dependencies_2.12-0.27.0.jar\"]\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "ee427d7c-edcb-46b2-a825-624db784aff0",
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Service Account:  617357862702-compute@developer.gserviceaccount.com\n"
+     ]
+    }
+   ],
+   "source": [
+    "SERVICE_ACCOUNT = \"\" \n",
```

Comment:
`SERVICE_ACCOUNT` has already been initialized in the previous cell

Reply:
i think this is part of the metadata of when the notebook gets run